app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb

```diff
@@ -21,6 +21,18 @@ def fetch_entities(client, entities)
             end
           end
         end
+
+        def fetch_ems_metrics(ems)
+          hawk = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(ems)
```

Comment:
@yaacov here I am creating a new HawkularClient for the ems, even though the ems already has a hawkular client for metrics. This is done once in every refresh for each provider. 
Is that a good practice? Do you see a better way to do it?